SymPEP-0001.md

```diff
@@ -0,0 +1,241 @@
+# SymPEP 1 --- SymPEP Purpose and Process
+
+**Author** Aaron Meurer, Jason K. Moore  
+**Status** Draft  
+**Type** Process  
+**Created** 2021-01-27  
+**Resolution**  
+
+## Abstract
+
+SymPEPs are a formal process whereby important changes are proposed and
+discussed. Corresponding to each SymPEP is a document that outlines the
+proposed change(s), the discussion around those changes, and motivation for
+that change. This document outlines the details of the SymPEP process and
+corresponding documents.
+
+## What is a SymPEP?
+
+SymPEP stands for "SymPy Enhancement Proposal". SymPEPs represent a formal
+process whereby important changes are proposed and discussed in the SymPy
+community. Corresponding to each SymPEP is a SymPEP document which outlines the
+proposal and corresponding discussion. SymPEPs are based on the
+[PEP](https://www.python.org/dev/peps/) process used by the Python language
+community to propose changes to the Python language. It also takes motivations
+from other similar processes in communities adjacent to SymPy, such as the
+[NEP](https://numpy.org/neps/nep-0000.html) process for NumPy and the
+[MEP](https://matplotlib.org/devel/MEP/index.html) process for Matplotlib.
+However, the SymPEP process differs from these processes in some ways, so those
+who may already be familiar with similar processes from other communities
+should read this document to understand how it works for SymPEPs.
+
+## Types
+
+There are three kinds of SymPEPs:
+
+A **Software Track** SymPEP describes a new feature or important change for the
+code in SymPy.
+
+A **Process** SymPEP describes a process related to SymPy, or proposes a
+change to (or an event in) a process. Process SymPEPs are like Software Track
+SymPEPs but apply to areas other than the SymPy library itself. They may
+propose an implementation, but not to SymPy’s codebase; they require community
+consensus. Examples include procedures, guidelines, changes to the
+decision-making process, and changes to the tools or environment used in SymPy
+development. Any meta-SymPEP is also considered a Process SymPEP.
+
+## Purpose
+
+The SymPEP process and corresponding documents serve several purposes:
+
+- To decide, as a community, whether a proposed change should be made.
+- To decide on the process by which a change will be implemented, for example,
+  whether the change should be implemented in several stages, whether
+  intermediate releases are required for things like deprecations, and so on.
+- To document the above discussions and decisions for future reference.
+- To document the motivations for a change from the perspective of end users
+  who may be affected by it. Here "end users" means both users of the SymPy
+  library, as well as developers working on parts of SymPy itself which may be
+  affected by the change.
```

Comment:
Another important group to mention here is developers of downstream libraries that depend on SymPy.